### PR TITLE
feat: eip712 clear sign on ledger, close #1816

### LINKED ISSUE #1816: Support clear-sign for EIP712 on Ledger
Ledger show only hashes when user prompted to sign EIP712 with Rabby and there is no easy way to verify what these hashes correspond to. Signing hashes is dangerous, because it might be phishy permit to transfer WETH, WBTC or monkey without expiration. Ledger supports displaying EIP712 domain and data on the device screen when signing with Ledger Live. Rabby should too! 
Might be related: https://pypi.org/project/eip712-clearsign/

### PR DESCRIPTION
**Pull request by heisenberg-2077**

### CODE REVIEW

**Comment by vvvvvv1vvvvvv** [comment]
Waiting for https://github.com/LedgerHQ/ledger-live/issues/5241 fixed

File: src/background/service/keyring/eth-ledger-bridge-keyring.ts [hunk 1]
```diff
@@ -718,11 +718,32 @@ class LedgerBridgeKeyring extends EventEmitter {
...
+            // https://github.com/LedgerHQ/ledger-live/blob/5bae039273beeeb02d8640d778fd7bf5f7fd3776/libs/coin-evm/src/hw-signMessage.ts#L68C7-L79C10
+            try {
+              res = await this.app!.signEIP712Message(hdPath, data);
+            } catch (e) {
+              if (
+                e instanceof Error &&
+                'statusText' in e &&
+                (e as any).statusText === 'INS_NOT_SUPPORTED'
+              ) {
+                res = await this.app!.signEIP712HashedMessage(
```
Comment (0x398) [inline src/background/service/keyring/eth-ledger-bridge-keyring.ts:738]: @heisenberg-2077 will it throw anyway on line 744 after this?

Comment (vvvvvv1vvvvvv) [reply src/background/service/keyring/eth-ledger-bridge-keyring.ts:738]: Yes, we need an `else` block here to avoid always throw the error

**Comment by 0x398** [comment]
> Waiting for [LedgerHQ/ledger-live#5241](https://github.com/LedgerHQ/ledger-live/issues/5241) fixed

Is bug in crypto-js really relevant for Ledger? It must be doing all sensitive stuff on the device anyway.

**Pull request merged**